Include/internal/pycore_frame.h

```diff
@@ -112,6 +112,7 @@ _PyFrame_InitializeSpecials(
     frame->prev_instr = _PyCode_CODE(code) - 1;
     frame->yield_offset = 0;
     frame->owner = FRAME_OWNED_BY_THREAD;
+    // frame->previous: initialized when frame is linked into the frame stack
```

Comment:
This looks a bit like commented out code.
Could you move the comment to above the function.

Reply:
Maybe change `/* Consumes reference to func and locals */` to 
```C
/* Consumes reference to func and locals.
   Does not initialize frame->previous, which happens
   when frame is linked into the frame stack.
*/
```